pycg/utils/common.py

```diff
@@ -19,24 +19,28 @@
 # under the License.
 #
 import os
+from typing import Optional
 
-def get_lambda_name(counter):
+
+def get_lambda_name(counter) -> str:
     return "<lambda{}>".format(counter)
 
-def get_dict_name(counter):
+
+def get_dict_name(counter) -> str:
     return "<dict{}>".format(counter)
 
-def get_list_name(counter):
+
+def get_list_name(counter) -> str:
     return "<list{}>".format(counter)
 
-def get_int_name(counter):
+
+def get_int_name(counter) -> str:
     return "<int{}>".format(counter)
 
-def join_ns(*args):
-    for arg in args:
-        if arg == None:
```

Comment:
I initially tried to type this as returning `Optional[str]` as per the original check here, which blew up into dozens of other places having to account for possible None. Then I removed this check, and I believe it is no longer necessary. Both MyPy and my own local testing suggests there's no situation in which an accidental None can make its way into this function.